index.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Ojwang</title>
+</head>
+
+<body>
+    <header class="toolbar">
+        <a href="#" class="logo">My Logo</a>
+        <nav class="nav-list">
+            <ul>
+                <li>
+                    <a href="#">Portfolio</a>
+                </li>
+                <li>
+                    <a href="#">About</a>
+                </li>
+                <li>
+                    <a href="#">Contact</a>
+                </li>
+            </ul>
+        </nav>
+        <div><a href="#" class="hamburger">&equiv;</a> </div>
+    </header>
+    <section class="hero">
+        <div class="hr-cont">
+            <h1 id="name">I'm Ojwang'.<br>Glad to see you!</h1>
+            <p class="description">I'm a software developer! I can help you build a product, feature or website. Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don't hestate to contact me.</p>
+            <div>
+                <h2 class="connect">LET'S CONNECT</h2>
+            </div>
+            <div class="icons">
+                <ul>
+                    <img src="images/twitter.svg" alt="">
+                </ul>
+                <ul>
+                    <img src="images/Linkedin.svg" alt="">
+                </ul>
+                <ul>
+                    <img src="images/Vector.svg" alt="">
+                </ul>
+                <ul>
+                    <img src="images/github.svg" alt="">
+                </ul>
+                <ul>
+                    <img src="images/peace.svg" alt="">
+                </ul>
```

Comment:
Let's learn how to use alt here, it should not be empty, `alt=""`, this is an empty alt attribute.

Kindly add text to all empty alt in your codebase.